pitest-maven-verification/src/test/java/org/pitest/PitMojoIT.java

```diff
@@ -398,32 +323,73 @@ private File buildFile(File base, String... pathParts) {
 	  StringBuilder path = new StringBuilder(base.getAbsolutePath());
 
 	  for (String part : pathParts) {
-		  path.append(File.separator).append(part);
+	      if(!StringUtils.isBlank(part)) {
```

Comment:
Would be an IllegalArgumentException more appropriatly? I think any blank is an illegal part of a path. Except I'd like to   hide sth.